app/frontend/src/components/apiAccess/Step4.vue

```diff
@@ -4,10 +4,8 @@
       <v-icon large color="green darken-2">check_circle</v-icon>API Documentation and Usage
     </h2>
     <p class="mb-12">
-      This service client can be used to make REST API calls and an example Postman collection is provided.
-      <br />Fetch a new token using the service client and password against the token endpoint (see postman collection) and use that token in your bearer header.
+      Your service client can make REST calls to a Common Services API. Please view the <router-link :to="{ name: 'Documentation' }">"Documentation".</router-link> page to learn more about these API's, get the full specifications and learn how to test them with Postman.
```

Comment:
Your service client can make REST calls to the Common Service APIs. Please view the Documentation to learn more about these APIs, their specifications, and how to test them with Postman.